e2e/playwright/testing-segment-overlays.spec.ts

```diff
@@ -90,15 +92,12 @@ test.describe('Testing segment overlays', () => {
         await expect(
           page.getByTestId('cmd-bar-arg-value').getByRole('textbox')
         ).toBeFocused()
+        await page.waitForTimeout(500)
         await page
           .getByRole('button', {
             name: 'arrow right Continue',
           })
           .click()
```

Comment:
There is a race in the cmdbar, if you click the progression too quickly, there's a error toast and it does nothing.

Not something that would be a big problem for users because they'de have to be really quick, plus clicking the button a second time and it's happy 🤷, but that's what this wait is for, wasn't sure how to properly wait for what ever the race is.